docs/code-samples/upgrades/daml/Extensions/MyTransferable.daml

```diff
@@ -0,0 +1,136 @@
+-- Copyright (c) 2023 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+-- SPDX-License-Identifier: Apache-2.0
+
+module Extensions.MyTransferable where
+
+import DA.Set (fromList, singleton)
+
+-- INTERFACE DEPENDENCIES --
+import Daml.Finance.Interface.Holding.Factory qualified as HoldingFactory (Create(..), I, View(..), holdingFactoryKey)
+import Daml.Finance.Interface.Holding.Holding qualified as Holding (I, View(..))
+import Daml.Finance.Interface.Holding.Transferable qualified as Transferable (I, View(..))
+import Daml.Finance.Interface.Types.Common.Types (AccountKey(..), Id, InstrumentKey(..), PartiesMap)
+import Daml.Finance.Interface.Types.Common.Types qualified as HoldingStandard (HoldingStandard(..))
+import Daml.Finance.Interface.Util.Disclosure qualified as Disclosure (I, View(..), flattenObservers)
+import Daml.Finance.Interface.Util.Lockable qualified as Lockable (I, Lock(..), View(..), getLockers)
+import Daml.Finance.Util.Disclosure (addObserversImpl, removeObserversImpl, setObserversImpl)
+import Daml.Finance.Util.Lockable (acquireImpl, isValidLock, releaseImpl)
+
+-- IMPLEMENTATION DEPENDENCIES --
+import Daml.Finance.Holding.BaseHolding (BaseHolding(..))
+import Daml.Finance.Holding.Fungible (Fungible(..))
+import Daml.Finance.Holding.TransferableFungible (TransferableFungible(..))
+import Daml.Finance.Holding.Util (transferImpl)
+
+-- | A new implementation of a holding which adheres to the `Transferable` holding standard.
```

Comment:
I would explain what makes this implementation different than the standard one

Reply:
I added a description.